portfolio/src/main/webapp/script.js

```diff
@@ -13,16 +13,36 @@
 // limitations under the License.
 
 /**
- * Adds a random greeting to the page.
+ * Adds a random picture of Luna to the page.
  */
-function addRandomGreeting() {
-  const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
+function randomLuna() {
+    const lunaIndex = Math.floor(Math.random() * 5);
```

Comment:
It may be preferable to use 'lunaNames.length' here instead of 5. That way if you ever add or remove images to this list in the future you don't also have to also remember to change this value. This'd require you to initialize lunaIndex after lunaNames.